dashboard/app/views/admin_reports/directory.html.haml

```diff
@@ -0,0 +1,46 @@
+%h1 Directory of Admin Dashboards
+
+%h3 Code Studio Progress
+= link_to admin_progress_path do
+  Admin Progress
+%br
+= link_to level_completions_path do
+  Level Completion
+%br
+= link_to pd_progress_path(:script_id => 33) do
+  PD Progress
+%br
+
+%h3 Hour of Code
+= link_to hoc_signups_path do
+  Event Signups
+%br
+-# Students Served
```

Comment:
Yep. It'll get enabled with the submission of the other PR. The placeholder is necessary since this doesn't pass tests (the link_to is an undefined route) without it.